locale/it/about/index.md

```diff
@@ -42,20 +42,20 @@ Se alcuni dei termini utilizzati non ti sono familiari, ecco un articolo complet
 
 ---
 
-Node è  influenzato da sistemi come Ru Ruby's
-[Event Machine][] o Python's [Twisted][]. Node prende un po' dal modello a eventi.
- Presenta un [event loop][] come un costrutto di runtime invece che come una libreria. In altri sistemi, c'è sempre una chiamata di blocco per avviare l'event-loop.
+Node.js è influenzato da sistemi come la [Event Machine][] di Ruby o [Twisted][] di Python.
+ Node.js prende un po' dal modello a eventi.
```

Comment:
I think it's better to remove this. It seems a little out of context.

Reply:
Is write on the English [page](https://nodejs.org/en/about/)

Reply:
Ok then I'd would prefer to use the original translation "Node porta il modello ad eventi un po' oltre." which better reflect the corresponding English sentence "Node takes the event model a bit further" in my opinion.

Reply:
You do the merge?

Reply:
Let's keep it open until tomorrow just in case someone wants to chime in. We can then merge it. Thank you for your patience.

Reply:
Ok, Thank you @lpinca.